client/src/components/Profile/ui/PrescreeningFeedback.tsx

```diff
@@ -3,65 +3,109 @@ import { StageInterviewDetailedFeedback } from '@common/models/profile';
 import { CODING_LEVELS, FeedbackStepId, SKILLS_LEVELS } from 'data/interviews/technical-screening';
 import { InterviewFeedbackStepData, InterviewFeedbackValues, InterviewQuestion } from '@common/models';
 import { Rating } from '@client/components/Rating';
+import React from 'react';
 
 const { Text, Title } = Typography;
 
-export function PrescreeningFeedback({ feedback }: { feedback: StageInterviewDetailedFeedback['feedback'] }) {
-  const { steps } = feedback as { steps: Record<FeedbackStepId, InterviewFeedbackStepData> };
+const STYLES = {
+  feedbackItemWidth: '80%',
+  skillCommentWidth: '60%',
+};
 
-  const { theory, practice, english, decision, intro } = steps;
-  const isRejected = intro.values?.interviewResult === 'missed';
+const FEEDBACK_CONFIG = [
```

Comment:
All items in this array have witdh `STYLES.feedbackItemWidth`. The `FeedbackItem` has the same default value for the `width` props. Can we safely skip it here?